core/src/commands/create/create-project.ts

```diff
@@ -138,30 +137,56 @@ export class CreateProjectCommand extends Command<CreateProjectArgs, CreateProje
 
       log.info("")
     }
-
-    let { yaml } = renderProjectConfigReference({
-      yamlOpts: {
-        onEmptyValue: "remove",
-        filterMarkdown: true,
-        renderBasicDescription: !opts["skip-comments"],
-        renderFullDescription: false,
-        renderValue: "preferExample",
-        presetValues: {
-          kind: "Project",
-          name,
-          apiVersion: GardenApiVersion.v1,
-          environments: [{ name: "default" }],
-          providers: [{ name: "local-kubernetes" }],
-        },
-      },
-    })
-
     const projectDocURL = `${DOCS_BASE_URL}/using-garden/projects`
     const projectReferenceURL = `${DOCS_BASE_URL}/reference/project-config`
-    yaml =
-      dedent`
+    const providersReferenceURL = `${DOCS_BASE_URL}/reference/providers`
+    const remoteK8sReferenceURL = `${DOCS_BASE_URL}/kubernetes-plugins/remote-k8s`
+    const actionsGettingStartedURL = `${DOCS_BASE_URL}/using-garden/actions`
+    const yaml = dedent`
     # Documentation about Garden projects can be found at ${projectDocURL}
-    # Reference for Garden projects can be found at ${projectReferenceURL}` + `\n\n${yaml}`
+    # Reference for Garden projects can be found at ${projectReferenceURL}
+
+    apiVersion: ${GardenApiVersion.v1}
+    kind: Project
+    name: ${name}
+
+    defaultEnvironment: local
+
+    # Environments typically represent different stages of your development and deployment process.
+    environments:
```

Comment:
I would love to also have a quick description of the local environment so it is more consistent e.g.:
```suggestion
    environments:
    
    # Use this environment to develop in your local Kubernetes solution of choice. For more information about
    # installation instructions and against which local Kubernetes solutions Garden is tested see:
    # ${localKubernetesInstallationURL}
```